firestore-bigquery-export/scripts/gen-schema-view/src/index.ts

```diff
@@ -41,86 +38,143 @@ const validateInput = (value: any, name: string, regex: RegExp) => {
   return true;
 };
 
+function collect(value, previous) {
+  return previous.concat([value]);
+}
+
+program
+  .name("gen-schema-views")
+  .option(
+    "--non-interactive",
+    "Parse all input from command line flags instead of prompting the caller.",
+    false
+  )
+  .option(
+    "-P, --project <project>",
+    "Firebase Project ID for project containing Cloud Firestore database."
+  )
+  .option(
+    "-d, --dataset <dataset>",
+    "The ID of the BigQuery dataset containing a raw Cloud Firestore document changelog."
+  )
+  .option(
+    "-t, --table-name-prefix <table-name-prefix>",
+    "A common prefix for the names of all views generated by this script."
+  )
+  .option(
+    "-f, --schema-files <schema-files>",
+    "A path in the filesystem specifying a globbed collection of files to read schemas from.",
+    collect,
+    []
+  );
+
 const questions = [
   {
     message: "What is your Firebase project ID?",
-    name: "projectId",
+    name: "project",
     type: "input",
     validate: (value) =>
       validateInput(value, "project ID", FIRESTORE_VALID_CHARACTERS),
   },
   {
     message:
       "What is the ID of the BigQuery dataset the raw changelog lives in? (The dataset and the raw changelog must already exist!)",
-    name: "datasetId",
+    name: "dataset",
     type: "input",
     validate: (value) =>
-      validateInput(value, "dataset", BIGQUERY_VALID_CHARACTERS),
+      validateInput(value, "dataset ID", BIGQUERY_VALID_CHARACTERS),
   },
   {
     message:
       "What is the name of the Cloud Firestore Collection that you would like to generate a schema view for?",
-    name: "collectionName",
+    name: "tableNamePrefix",
```

Comment:
NIT:
... name of the Cloud Firestore collection for which you want to generate a schema view?